server/autolinkplugin/config_test.go

```diff
@@ -0,0 +1,60 @@
+package autolinkplugin
+
+import (
+	"errors"
+	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin/plugintest"
+	"github.com/mattermost/mattermost-server/v5/plugin/plugintest/mock"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestOnConfigurationChangeInvalidConfig(t *testing.T) {
```

Comment:
Thanks for add a test :+1: 

Nit pick: If you have similar tests like `TestOnConfigurationChangeInvalidConfig` and `TestOnConfigurationChangeInvalidLink` you can group them by using [Subtests](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks). This is more of can FYI then a request for change.